components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/CertificateValidationUtil.java

```diff
@@ -206,6 +225,12 @@ public static void loadCRLDownloadTimeoutFromConfig() {
     public static List<RevocationValidator> loadEnabledValidatorConfigFromRegistry()
             throws CertificateValidationException {
 
+        return getRevocationValidatorsFromConfigStore();
+    }
+
+    private static List<RevocationValidator> getRevocationValidatorsFromRegistry()
```

Comment:
Is there any usage of this private method?

Reply:
Not at this moment.. But this this method may have a usage depending on the fallback config approach we need to implement as the next step. That part is not finalized yet.. Depending on that, either this method will have usage or this method will be removed.